{
  "comments": [
    {
      "key": {
        "uuid": "5a3f45e8_c3cf6faf",
        "filename": "api/resources/v1/env.py",
        "patchSetId": 1
      },
      "lineNbr": 88,
      "author": {
        "id": 7075
      },
      "writtenOn": "2018-02-27T12:07:53Z",
      "side": 1,
      "message": "Only one exception will be raisen and catch. \"errors\" is not going to be a list, but an instance of APIError or HTTPError.",
      "range": {
        "startLine": 88,
        "startChar": 12,
        "endLine": 88,
        "endChar": 28
      },
      "revId": "d05b101d37355ae151e10fcf4ab398de3d7ad667",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "5a3f45e8_23c45391",
        "filename": "api/resources/v1/env.py",
        "patchSetId": 1
      },
      "lineNbr": 294,
      "author": {
        "id": 7075
      },
      "writtenOn": "2018-02-27T12:07:53Z",
      "side": 1,
      "message": "ditto",
      "range": {
        "startLine": 294,
        "startChar": 13,
        "endLine": 294,
        "endChar": 28
      },
      "revId": "d05b101d37355ae151e10fcf4ab398de3d7ad667",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}